### PR TITLE
[PF-1747] Install pandas_gbq in notebook startup script

### PR DESCRIPTION
**Pull request by melissachang**

### CODE REVIEW

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh [hunk 1]
```diff
@@ -88,6 +88,8 @@ if [[ -n "$TERRA_SSH_KEY" ]]; then
   sudo -u "$JUPYTER_USER" sh -c 'ssh-add .ssh/id_rsa; ssh-keyscan -H github.com >> ~/.ssh/known_hosts'
 fi
 
+sudo -u "${JUPYTER_USER}" sh -c "pip install pandas_gbq"
```
Comment (zloery) [inline service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:91]: I don't see any other `pip` commands in this script - just to confirm, do notebooks have `pip` installed already?

**Review by zloery — APPROVED** [review]

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh (hunk 1 above)
Comment (melissachang) [reply service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:91]: I just created a notebook, waited for startup script to finish, and ran `pip install pandas_gbq`. It worked.

FYI - default Vertex AI image also has some python packages preinstalled. They have `pandas` preinstalled, but not `pandas_gbq`.

Comment (yuhuyoyo) [inline service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh:91]: nit: can we do it next to other common package installation.

**Review by yuhuyoyo — APPROVED** [review]

File: service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh (hunk 1 above)
Comment (melissachang) [reply service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/post-startup.sh]: Ah, I missed the `conda install`. Moved, thanks.

**Comment by yuhuyoyo** [comment]
it's already broken whether it's merged downstream or not. We are setting the default post-startup.sh  as the upstream repo raw file urll: https://github.com/DataBiosphere/terra-workspace-manager/blob/f1d56f41fc92251df0bc240d88c04e3d914d3933/service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/CreateAiNotebookInstanceStep.java#L58 @melissachang

**Pull request merged**